src/Chainweb/Pact/PactService.hs

```diff
@@ -637,10 +648,31 @@ playOneBlock
 playOneBlock currHeader plData pdbenv = do
     miner <- decodeStrictOrThrow' (_minerData $ _payloadDataMiner plData)
     trans <- liftIO $ transactionsFromPayload plData
+
+    mapM_ checkTx trans
     !results <- go miner trans
     psStateValidated .= Just currHeader
     return $! toPayloadWithOutputs miner results
   where
+    checkTTL = if isGenesisBlock then const $ const True else timingsCheck
```

Comment:
Add a TODO entry to remind ourselves that we need to dupe check in here. actually we should just do exactly the same newblock checks and delete all the duplicated code, but let's fix that later

Reply:
done